src/index.html

```diff
@@ -4,9 +4,367 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/main.css">
```

Comment:
Use here `main.scss`
```suggestion
    <link rel="stylesheet" href="./styles/main.scss">
```